src/domains/transaction/components/VoteDetail/VoteDetail.tsx

```diff
@@ -57,19 +57,13 @@ export const VoteDetail = ({ transaction, walletAlias, ticker, isOpen, onClose }
 	const [delegates, setDelegates] = useState<ReadOnlyWallet[]>([]);
 
 	useEffect(() => {
-		const syncDelegates = async () => {
+		const syncDelegates = () => {
 			setIsLoadingDelegates(true);
 
-			// TODO: make senderWallet non-nullable
```

Comment:
`senderWallet` is still nullable.

Reply:
Yeah, didn't touch vote details to fix this part yet, getting the background sync working first, gonna update this before the final PR 🚀